traits/observers/_i_observable.py

```diff
@@ -0,0 +1,29 @@
+# (C) Copyright 2005-2020 Enthought, Inc., Austin, TX
+# All rights reserved.
+#
+# This software is provided without warranty under the terms of the BSD
+# license included in LICENSE.txt and may be redistributed only under
+# the conditions described in the aforementioned license. The license
+# is also available online at http://www.enthought.com/licenses/BSD.txt
+#
+# Thanks for using Enthought open source!
+
+
+class IObservable:
+    """ Interface for objects that can emit notifications for the observer
+    system.
+    """
+
+    def _notifiers(self, force_create):
```

Comment:
Alternative to this ugly `force_create`: Create a method of a different name with no argument and have `CTrait` implements this new method.

Reply:
`_notifiers` is private, and only used internally to Traits, so we can also change its behaviour in `ctraits.c` if that's the most practical option.

Reply:
My motivation there was purely cosmetic: It is a bit of an eyesore everywhere when I implement the interface of `IObservable` I need to document this `force_create` not being used. I am happy to delay this though cosmetic change. The interface is also for internal use only.